tsconfig.json

```diff
@@ -24,6 +24,6 @@
         "skipDefaultLibCheck": true, // Skip type checking .d.ts files that are included with TypeScript
         "downlevelIteration": true
     },
-    "include": ["utils.d.ts"],
+    "files": ["./types/global.d.ts", "./types/utils.d.ts", "./types/svg.d.ts"],
```

Comment:
@hamid-deriv Nice 🚀
Would you please remove them from all the packages `tsconfig.json` file 🙏🏻

Reply:
Done